src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,33 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post, users, comments }) => {
+  const author = users.find(user => post.userId === user.id);
+  const arrOfComments = comments.filter(
+    comment => comment.postId === post.id,
+  );
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{post.title}</h3>
+
+        <p>
+          {`Posted by `}
+          <UserInfo user={author} />
```

Comment:
author might be `undefined` (`find` method might return `undefined` theoretically) - you should handle this case